integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -176,7 +177,9 @@ private int createOrOpenInternal(String path, FuseFileInfo fi, long mode) {
       return res;
     }
     try {
-      FuseFileStream stream = mStreamFactory.create(uri, fi.flags.get(), mode);
+      FuseFileStream stream = DoraCacheFileSystem.class.isInstance(mFileSystem)
```

Comment:
can put in the PathResolver, i can deal with this part in later PR

Reply:
Let me explain the problems here.

`mFileSystem` might be DoraCacheFileSystem, or UfsBaseFileSystem.
This class is used in both UFS and Dora file system tests.
The file path in DoraCacheFileSystem is Alluxio based file path and,
the file path in UfsBaseFileSystem is absolute/full path.
Let's assume a `path` passed to this function is "/Some_Dir/Some_File".
Let's assume we are using:
alluxio.master.mount.table.root.ufs=/Users/xxx/alluxio/underFSStorage

In Dora test, the final file in host OS is:
/Users/xxx/alluxio/underFSStorage/Some_Dir/Some_File
In UfsBaseFileSystem test, the final file in host OS is:
/Some_Dir/Some_File

Before the test running, the test suite will create a file and write some data into it.
When the test is running, we will check the file.
ERROR will happen now, because in preparation, the `/Some_Dir/Some_File` is created in host OS.
But in Dora test, the expected file in host OS will be: `/Users/xxx/alluxio/underFSStorage/Some_Dir/Some_File`
This causes tests error.

That's why I am checking if the `mFileSystem` is Dora or not.
This is purely a bug/problem in test suite. Not in the Alluxio main code.